src/reference-assemblies/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
`DefaultTargets="Build"` is unnecessary.  This .props file doesn't contain any targets.

(Technically `ToolsVersion` and `xmlns` are unnecessary as well with MSBuild v15)

Reply:
Removed it.